lib/settings/wifi_screen.dart

```diff
@@ -134,20 +201,27 @@ class _WifiScreenState extends State<WifiScreen> {
           final RegExpMatch? match = pattern.firstMatch(lines[i]);
 
           if (match != null) {
-            /*if (kDebugMode) {
-              print('---------------------------');
-              for (int i = 1; i < match.groupCount; i++) {
-                print('Group $i: ${match.group(i)}');
-              }
+            /*print('---------------------------');
+            for (int i = 1; i < match.groupCount; i++) {
+              print('Group $i: ${match.group(i)}');
             }*/
```

Comment:
and this lol. This could mvoe to a debug log call if necessary

Reply:
Resolved by [17dd19d](https://github.com/TheContrappostoShop/Orion/pull/14/commits/17dd19d100d306003192341c4a4ca3d23aad545d)